packages/test/test-utils/src/containerUtils.ts

```diff
@@ -89,3 +97,16 @@ export async function getDataStoreEntryPointBackCompat<T>(dataStore: IDataStore)
 	assert(response.status === 200, "empty request should return data object");
 	return response.value as T;
 }
+
+/**
+ * @internal
+ */
+export function assertIsIDeltaManagerFull(
```

Comment:
naming nit: I would expect `assertIs*` to assert and return void.
An alternate name would be `toIDeltaManagerFullAsserted` though I think we'd also be fine with just `toIDeltaManagerFull`.